data-structures-stacks/get-top.js

```diff
@@ -0,0 +1,8 @@
+/* exported getTop */
+
+function getTop(stack) {
+  if (!stack.print()) {
+    return undefined;
+  } else stack.peek();
+  return stack.peek();
```

Comment:
You really only need this last line, if there is no items `peek` will return `undefined` so the end result will be the same.